src/jquery.nanogallery2.core.js

```diff
@@ -1485,6 +1485,7 @@
 		viewerGallery:								'bottomOverMedia',
 		viewerGalleryTWidth:					40,
 		viewerGalleryTHeight:  				40,
+    viewerGalleryHide:            false,

```

Comment:
I might be looking at this wrong, but it seems liek your indentation is way off from the above text. Would you be willing to standardize so the file looks a little more uniform?